alarms/alarms.py

```diff
@@ -60,6 +60,27 @@
 MARKER_SIMPLIFICATION = re.compile(r'{(\w*):?.*?}')
 
 
+def read_settings():
+    settings = {}
+    expected_elements = (
+        'smtp_host', 'db_host', 'log_file_name', 'db_name',
+        'reader_user', 'reader_pw', 'alarm_user', 'alarm_pw',
+        'error_contact_email', 'mail_sender'
+    )
+    system_global = xml.etree.ElementTree.ElementTree()
+    conf = pathlib.Path(__file__).resolve().parent.parent / 'global_settings.xml'
+    system_global.parse(conf)
+    system_global = system_global.getroot()
+
+    for child in system_global:
+        if child.tag == 'mail_alarm_settings':
+            break
+    for element in child:
+        if element.tag in expected_elements:
```

Comment:
Also don't know if this is a real problem, but if "expected" means mandatory, then there is no check for completeness of the configuration here. Could do something like:

```python
    elements_left_to_find = set(expected_elements)
    for element in child:
        if element.tag in elements_left_to_find:
            elements_left_to_find.pop(element.tag)
            settings[element.tag] = element.text

    if elements_left_to_find:
        raise ValueError(f"Alarms configuration missing keys: {elements_left_to_find}")
```

This error might be easier to parse than the error one might get if a mandatory settings was missing.

Reply:
I agree that the settings check could use some more details, but I am fine with adding this in a later PR. (Maybe add a TODO for placeholder if you don't want to look at it now

Reply:
Agreed, I think this can be postpone for a release focus on more robust deployment.